sdcm/cluster.py

```diff
@@ -733,15 +733,11 @@ def _get_n_coredumps(self):
         :return: Number of coredumps
         :rtype: int
         """
-        try:
-            n_backtraces_cmd = 'sudo coredumpctl --no-pager --no-legend 2>/dev/null'
-            result = self.remoter.run(n_backtraces_cmd,
-                                      verbose=False, ignore_status=True)
-            return len(result.stdout.splitlines())
-        except Exception as details:
-            self.log.error('Error retrieving number of core dumps : %s',
-                           details)
-            return None
+        n_backtraces_cmd = 'sudo coredumpctl --no-pager --no-legend 2>&1 ||true'
```

Comment:
We have ignore_status=True in run(), why do we still need the `|| true`?

Reply:
`ignore_status=True` means don't throw exception, this PR is about not to printing error message when command failed (currently if the command fails we print error message any way, whether verbose argument was True or False)